homeassistant/components/homekit/type_cameras.py

```diff
@@ -164,7 +164,7 @@ def __init__(self, hass, driver, name, entity_id, aid, config):
             },
             "resolutions": resolutions,
         }
-        audio_options = {"codecs": [{"type": "OPUS", "samplerate": 24}]}
+        audio_options = {"codecs": [{"type": "OPUS", "samplerate": 16}]}
```

Comment:
I think we need to expose both 24 and 16.


```suggestion
        audio_options = {"codecs": [{"type": "OPUS", "samplerate": 24}, {"type": "OPUS", "samplerate": 16}]}
```

Need to see how this works with the audio codec set to `copy`
https://www.home-assistant.io/integrations/homekit/#audio_codec

Reply:
Ok we can use both. I have set one camera to `copy` and both devices still work as expected